terra-jupyter-base/Dockerfile

```diff
@@ -48,15 +49,15 @@ RUN apt-get update && apt-get install -yq --no-install-recommends \
     libbz2-dev \
     libreadline-dev \
     libsqlite3-dev \
+    libexempi3 \
```

Comment:
Same here, does this require a comment?

Reply:
oh lol this change was a result of me removing and re-adding, nothing's changed these 2 packages are just further up in the list